media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="dav1d is a new AV1 cross-platform decoder, open-source, \
+and focused on speed and correctness. It supports all features from AV1, \
+including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64 ?arm ?ppc"
+
+PROVIDES="
+	dav1d = $portVersion
+	lib:libdav1d = 3.1.0 compat >= 3
+	cmd:dav1d
+	"
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	dav1d_devel = $portVersion
+	devel:libdav1d = 3.1.0 compat >= 3
+	"
+REQUIRES_devel="
+	dav1d == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:meson >= 0.47
+	cmd:ninja
+	cmd:doxygen
+	cmd:nasm >= 2.13.02
+	"
+
+defineDebugInfoPackage dav1d \
+	$binDir/dav1d
+
+BUILD()
+{
+	rm -f -rf build
+	mkdir build && cd build
+
+	meson .. --prefix=$prefix
+
+	ninja $jobArgs
+
+	doxygen doc/Doxyfile
+}
+
+INSTALL()
+{
+	ninja -C build install
+
+	#Manually copying some files to the right places
+	mkdir -p {$includeDir/dav1d,$documentationDir/dav1d}
+
+	mv -v $prefix/include/dav1d/* $includeDir/dav1d
+	prepareInstalledDevelLibs libdav1d
+	packageEntries devel $includeDir
+	packageEntries devel $developLibDir
+	rm -v -rf $prefix/include/*
+	rmdir -v $prefix/include
```

Comment:
Not sure what you are doing here, but doesn't really look OK

Reply:
The `ninja install` command copies the header files in a folder called "include" in the main package, which should not be done.
The lines 
```
	#empty the directory, so rmdir can finish the work.
	rm -v -rf $prefix/include/* 
	rmdir -v $prefix/include
```
cleans up the include folder (all its important contents are already moved to `$includeDir`